src/components/pages/heroarea.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import BackButton from './backbutton';
+
+const Header = ({ backButtonTitle, title }) => (
+  <div className="header-container">
+    <div className="header-contents">
+      <BackButton title={backButtonTitle} />
+      <p className="back-button">{title}</p>
+    </div>
+  </div>
```

Comment:
- [ ] Your design does not match the given one. Kindly add all the icons in the navigation bar even if they are not functional.

Reply:
![project](https://user-images.githubusercontent.com/77216006/134396350-9194f6a7-b0c2-4e68-9b67-b942a64aad07.png)

This is the button.